mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -314,6 +315,20 @@ class Word2Vec extends Serializable with Logging {
     val expTable = sc.broadcast(createExpTable())
     val bcVocab = sc.broadcast(vocab)
     val bcVocabHash = sc.broadcast(vocabHash)
+      do_fit(dataset, sc, expTable, bcVocab, bcVocabHash)
```

Comment:
Are we missing a try? The formatting should be

```
try {
  ...
} finally {
  ...
}
```

Also use camelCase rather than underscore_naming